spec/controllers/two_factor_authentication/otp_verification_controller_spec.rb

```diff
@@ -107,10 +107,16 @@
           enabled_mfa_methods_count: 1,
           in_multi_mfa_selection_flow: false,
         }
+
         stub_analytics
+        stub_attempts_tracker
+
         expect(@analytics).to receive(:track_mfa_submit_event).
           with(properties)
 
+        expect(@irs_attempts_api_tracker).to receive(:mfa_verify_phone_otp_submitted).
+          with({ reauthentication: false, success: false })
```

Comment:
we can drop the optional braces here I think
```suggestion
          with(reauthentication: false, success: false)
```